compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/validator/errors/ErrorMessage.java

```diff
@@ -43,7 +43,10 @@
     ERROR_115("A GraphQL field cannot use an output type as an input type"),
     ERROR_116("The graphql:Context should be the first parameter"),
     ERROR_117("Path parameters not allowed in GraphQL resources"),
-    ERROR_118("A GraphQL resource must have a name");
+    ERROR_118("A GraphQL resource must have a name"),
+    ERROR_119("The graphql:FileUpload cannot be used as an input type of resource function"),
+    ERROR_120("The graphql:FileUpload cannot be used as a return type"),
+    ERROR_121("The graphql:FileUpload cannot be used as field type");
```

Comment:
Do we need these two validations? The `graphql:FileUpload` record has a `stream` type field, which is not allowed in the GraphQL services. So it will be an error, right?
(Better to test this anyway).